Classic resources do not support tags

vivekr20 opened the pull request:
Adding a line to the limits section to call out that classic resources do not support tagging.

azurecla commented:
Hi __@sahaj08__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sahajs). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess

vivekr20 committed:
Merge pull request #1 from MicrosoftDocs/master

Merging latest origin master with my fork

vivekr20 committed:
classic resources do not support tags

adding a line to the limits section to call out that classic resources do not support tagging.
